src/Integration/TelemetryLoggerAccessor.cs

```diff
@@ -17,7 +18,7 @@ public static ITelemetryLogger GetLogger(IServiceProvider serviceProvider)
             IComponentModel componentModel = serviceProvider.GetService(typeof(SComponentModel)) as IComponentModel;
             if (componentModel != null)
             {
-                return componentModel.GetService<ITelemetryLogger>();
+                return componentModel.GetExtensions<ITelemetryLogger>().SingleOrDefault();
```

Comment:
Avoided updating the tests by injecting another MEF component. I manually tested that the telemetry is working instead.